onboarding-app/CounterViewController.swift

```diff
@@ -0,0 +1,70 @@
+import Foundation
+import UIKit
+
+class CounterViewController: UIViewController {
```

Comment:
we make classes final which are not intended to be subclassed. Adding final keyword to class does magic. Find that magic and let me know.